projects/igniteui-angular/src/lib/splitter/splitter-bar/splitter-bar.component.html

```diff
@@ -1,15 +1,15 @@
 <div class="igx-splitter__bar" [style.flex-direction]="direction" igxDrag [ghost]="false"
 (dragStart)='onDragStart($event)'
 (dragMove)="onDragMove($event)">
-    <div class="igx-splitter-bar-expander">
-        <igx-icon fontSet="material">{{collapsePrevIcon}}</igx-icon>
-    </div>    
+    <div class="igx-splitter-bar-expander" (click)='onCollapsing($event)'>
```

Comment:
you can just send false/true here so you don't have to extract the clicked button in the handler